.github/workflows/windows-app-test.yml

```diff
@@ -16,20 +16,11 @@ jobs:
         node-version: '12.9.1'
 
     - name: Setup MSBuild
-      uses: microsoft/setup-msbuild@v1.0.0
+      uses: microsoft/setup-msbuild@v1
       with:
         vs-version: 16.5
 
-    - name: Check node modules cache
-      uses: actions/cache@v1
-      id: yarn-cache
-      with:
-        path: ./example/node_modules
-        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
```

Comment:
so strange this does not work, as the key should miss and thus not restore
I have good luck with this incantation:

https://github.com/invertase/react-native-firebase/blob/63b822cd404892a61403c1c4c10d91a8ec4f13cc/.github/workflows/tests_e2e.yml#L56-L65

I think the caching should be a net positive, not least for speed, but because it will by definition fetch from network less and my largest sources of CI flakiness in the repos I manage are transient network failures fetching remote resources needed to do the CI.

Because of that, may I ask you to try the way we do it on react-native-firebase to see if it works? I guess this won't be easy to reproduce though it seemed that it relies on a 0.62 artifact to be built and only then fails when building 0.63?

I'm prepared to accept this if you don't have a good / easy way to reproduce it to check, and will just re-investigate if it turns into a source of CI flakiness

Reply:
I think I see the original issue now. It's caching the `./example/node_modules` path but is using the `./yarn.lock` file instead of the `./example/yarn.lock` file.
Funny enough, I don't think this gains much speed. On previous runs I noticed that cache downloading and modules installing both took ~5minutes. The CI flakiness point is very valid though.
I'm going to split into two caches to solve the issue.

Reply:
Nevermind, there's time saving after all, from my tests.